common/src/test/java/com/alibaba/nacos/common/notify/NotifyCenterTest.java

```diff
@@ -0,0 +1,538 @@
+/*
+ * Copyright 1999-2018 Alibaba Group Holding Ltd.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.alibaba.nacos.common.notify;
+
+import com.alibaba.nacos.common.notify.listener.SmartSubscriber;
+import com.alibaba.nacos.common.notify.listener.Subscriber;
+import com.alibaba.nacos.common.utils.ThreadUtils;
+import org.junit.Assert;
+import org.junit.FixMethodOrder;
+import org.junit.Test;
+import org.junit.runners.MethodSorters;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.concurrent.CountDownLatch;
+import java.util.concurrent.TimeUnit;
+import java.util.concurrent.atomic.AtomicInteger;
+import java.util.concurrent.atomic.AtomicLong;
+
+/**
+ * @author <a href="mailto:liaochuntao@live.com">liaochuntao</a>
+ * @author zongtanghu
+ */
+@FixMethodOrder(value = MethodSorters.NAME_ASCENDING)
+public class NotifyCenterTest {
+    
+    private static class TestSlowEvent extends SlowEvent {
+    }
+    
+    private static class TestEvent extends Event {
+        
+        @Override
+        public long sequence() {
+            return System.currentTimeMillis();
+        }
+    }
+    
+    static {
+        System.setProperty("nacos.core.notify.share-buffer-size", "8");
+    }
+    
+    @Test
+    public void test_a_event_can_listen() throws Exception {
```

Comment:
ut test method also use Camel Naming

Reply:
Yeah, I have already renamed these ut test method name in next commit, please review again, thanks.